### DISCUSSION TITLE
Reolink flv substream

### DISCUSSION THREAD

**Discussion by pixbalance** [blakeblackshear/frigate#9472] [Ask A Question] [answered]
I am using the from suggestion for the reolink cameras for the flv stream. 

`room:  "ffmpeg:http://x.x.x.x/flv?port=1935&app=bcs&stream=channel0_main.bcs&user=usr&password=pwd#video=copy#audio=copy#audio=opus"`
`room_sub:  "ffmpeg:http://x.x.x.x/flv?port=1935&app=bcs&stream=channel0_ext.bcs&user=usr&password=pwd"`

I have set the high stream to 2560x1440 and the low stream to 640x360 in the cam. If I copy the the above links to VLC the main streams matches the high stream size but the sub stream is 896x512 instead of 640x360.

Did I miss something to change or is this fine? If it is fine, what size should I use in detect? 896x512 or 640x360?

**Comment by jurasjo** [comment, marked as answer]
Hi, use _sub instead of the _ext in the links. I believe Reolinks have actually 3 streams.

**Reply by pixbalance** [reply]
great! that's it.
